awxkit/awxkit/scripts/basic_session.py

```diff
@@ -59,11 +59,8 @@ def main():
 
         global root
         root = api.Api()
-        if uses_sessions(root.connection):
-            config.use_sessions = True
-            root.load_session().get()
-        else:
-            root.load_authtoken().get()
+        config.use_sessions = True
+        root.load_session().get()
```

Comment:
I'm going to revert this change, because it is the mostly likely candidate for the stuff that has been going wrong in testing. Using sessions vs not is still a valid thing, and this is probably mistaken.